http_endpoints.go

```diff
@@ -75,6 +76,11 @@ func Track() gin.HandlerFunc {
 
 		event := getEventFromBody(c)
 		for _, e := range event.Events {
+			if e.MetaData == nil {
+				e.MetaData = make(map[string]interface{})
+			}
+			e.MetaData["sdkProxy"] = Version
+			fmt.Println("event", e)
```

Comment:
```suggestion
```

Reply:
fixed, will manually test this once again for custom events